src/test/java/uk/nhs/hee/tis/trainee/sync/config/MongoConfigurationTest.java

```diff
@@ -108,13 +108,13 @@ void shouldInitIndexesForProgrammeMembershipCollection() {
     verify(indexOperations, atLeastOnce()).ensureIndex(indexCaptor.capture());
 
     List<IndexDefinition> indexes = indexCaptor.getAllValues();
-    assertThat("Unexpected number of indexes.", indexes.size(), is(4));
+    assertThat("Unexpected number of indexes.", indexes.size(), is(3));
```

Comment:
this will need to change back to 4 if we create a new programmeMembershipCompoundIndex with a different name